components/Discovery.js

```diff
@@ -80,73 +29,45 @@ const fetcher = (url, opts) => {
   }).then(d => d.json())
 }
 
-export const Discovery = ({network, appVersion, extensions, walletInclude, handleCancel}) => {
+export const Discovery = ({network, appVersion, extensions, walletInclude}) => {
```

Comment:
Maybe we can grab extensions off of config on next release, once we merge the account-proof-v2 stuff

Reply:
@gregsantos Definitely can